api/users/serializers.py

```diff
@@ -252,3 +252,17 @@ def get_absolute_url(self, obj):
 
     class Meta:
         type_ = 'institutions'
+
+
+class UserIdentitiesSerializer(BaseAPISerializer):
```

Comment:
We follow the [JSONAPI specification](http://jsonapi.org/) to format our API responses, and your response here does not follow the spec.  (The ticket was a little confusing here, and I can see why it made you think the data should be formatted this way). If you inherit from the `JSONAPISerializer` instead of `BaseAPISerializer` a lot of this is taken care of for you.  

My recommendation, provided you change your views as well:
```
class UserIdentitiesSerializer(JSONAPISerializer):
    id = IDField(source='_id', read_only=True)
    type = TypeField()
    external_id = ser.CharField(read_only=True)
    status = ser.CharField(read_only=True)

    links = LinksField({
        'self': 'get_absolute_url',
    })

    def get_absolute_url(self, obj):
        return absolute_reverse(
            'users:user-identities-detail',
            kwargs={
                'user_id': self.context['request'].parser_context['kwargs']['user_id'],
                'version': self.context['request'].parser_context['kwargs']['version'],
                'identity_id': obj['_id']
            }
        )

    class Meta:
        type_ = 'external-identities'

```